src/ray/core_worker/task_manager.cc

```diff
@@ -1015,7 +1015,7 @@ bool TaskManager::RetryTaskIfPossible(const TaskID &task_id,
     // TODO(clarng): clean up and remove task_retry_delay_ms that is relied
     // on by some tests.
     int32_t delay_ms = task_failed_due_to_oom
-                           ? ExponentialBackoff::GetBackoffMs(
+                           ? ExponentialBackOff::GetBackoffMs(
```

Comment:
Nit: backoff is one word (https://en.wikipedia.org/wiki/Exponential_backoff) so we should use `ExponentialBackoff`

Reply:
sure updated